test/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+[sh_test(
+    name = test,
+    # top showed me 50-400mByte of usage, so "enormous"...
+    #
+    # https://bazel.build/reference/be/common-definitions#test.size
+    size = "enormous",
+    srcs = ["bazel_test.sh"],
+    args = [],
+    data = [
+        "//:openroad",
+    ]
+    # TODO Refine blob later.
+    #
+    # This is a very broad blob, but most of the data in size
+    # comes from .lib files and we can't know which pdk files
+    # to include for a test given the name only.
+     + glob(["**/*"]),
+    env = {
+        "TEST_NAME_BAZEL": test,
+        "OPENROAD_EXE": "$(location //:openroad)",
+        "TEST_EXT": "tcl",
+    },
+    # FIXME bazel test ... should run these tests in optimized mode only
```

Comment:
@QuantamHD do you know if there is a way to only enable these tests on `bazel test ...` when `-c opt` is enabled ?

Reply:
No there's not. We shouldn't try to do that either. That's really a CI related decision

Reply:
Maybe https://github.com/The-OpenROAD-Project/OpenROAD/pull/6963 ?